content/en/docs/components/training/overview.md

```diff
@@ -1,79 +1,89 @@
 +++
 title = "Overview"
-description = "An overview of the Training Operator"
+description = "An overview of the Kubeflow Training"
```

Comment:
```suggestion
description = "An overview of Kubeflow Training"
```
or THE Kubeflow Training... API?

Reply:
Kubeflow Training is the official product/project name.
@vsoch @franciscojavierarceo @rimolive @varodrig @hbelmiro @kubeflow/wg-training-leads @thesuperzapper  Should we use the definite article everywhere or not ?